src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +14,7 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        return (a ? 1 : 0) + (b ? 1 : 0) + (c ? 1 : 0) + (d ? 1 : 0) == 2;
```

Comment:
smart solution, however main goal of this task is to practice with usage of logical operators (||, && and ^), so i would advice solve it with different approach🙂

Reply:
Okay, 1 minute. I know solution